cif_pow.dic

```diff
@@ -121,6 +121,1229 @@ save_pd_amorphous.phase_id
+    representing the background in a calculated diffractogram. Must be given
+    with a _pd_background.Chebyshev_order value.
+
+    The Chebyshev polynomial is of the first kind, and can be represented by the
+    recurrence relation
+
+        T~0~(x) = 1
+        T~1~(x) = x
+        T~n+1~(x) = 2 * x * T~n~(x) - T~n-1~(x).
+
+    where n represents the order of the polynomial, and x is the X-coordinate in
+    which the diffractogram was calculated, normalised to the range -1:1.
+
+    The background equation using Chebyshev polynomials is of the form:
+
+        bkg = Sum( coeff * T~n~(x))
+;
+    _name.category_id             pd_background
+    _name.object_id               Chebyshev_coeff
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   none
+
+save_
+
+save_pd_background.Chebyshev_coeff_su
+
+    _definition.id                '_pd_background.Chebyshev_coeff_su'
+    _definition.update            2023-01-08
+    _description.text
+;
+    Standard uncertainty of pd_background.Chebyshev_coeff.
+;
+    _name.category_id             pd_background
+    _name.object_id               Chebyshev_coeff_su
+    _name.linked_item_id          '_pd_background.Chebyshev_coeff'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_pd_background.Chebyshev_coeff_list
+
+    _definition.id                '_pd_background.Chebyshev_coeff_list'
+    _definition.update            2023-01-08
+    _description.text
+;
+    List of coefficients used in a Chebyshev equation representing the
+    background in a calculated diffractogram.
+
+    See _pd_background.Chebyshev_coeff.
+
+    The position of the coefficient in the list is significant, denoting the
+    order to which it corresponds. The first is the zeroth order, the second
+    is the first, and so on.
+;
+    _name.category_id             pd_background
+    _name.object_id               Chebyshev_coeff_list
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Matrix
+    _type.dimension               '[]'
+    _type.contents                Real
+    _units.code                   none
+
+save_
+
+save_pd_background.Chebyshev_coeff_list_su
+
+    _definition.id                '_pd_background.Chebyshev_coeff_list_su'
+    _definition.update            2023-01-08
+    _description.text
+;
+    Standard uncertainty of pd_background.Chebyshev_coeff_list.
+;
+    _name.category_id             pd_background
+    _name.object_id               Chebyshev_coeff_list_su
+    _name.linked_item_id          '_pd_background.Chebyshev_coeff_list'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
```

Comment:
The import here will end up giving the SU as having a `Single` container instead of `Matrix`.

Reply:
ah! A trip for the unwary writer.